130-kernel module no longer exists

### Issue Description

```
Run ./vib build recipe.yml
Downloading recipe from gh:Vanilla-OS/core-image:main:modules/05-firmware.yml
Downloading recipe from gh:Vanilla-OS/core-image:main:modules/10-input-and-locale.yml
Downloading recipe from gh:Vanilla-OS/core-image:main:modules/130-kernel.yml
Downloading recipe from gh:Vanilla-OS/core-image:main:modules/50-fs.yml
Downloading recipe from gh:Vanilla-OS/core-image:main:modules/70-compression.yml
Downloading recipe from gh:Vanilla-OS/core-image:main:modules/90-network.yml
Downloading recipe from gh:Vanilla-OS/core-image:main:modules/91-iptables.yml
Building recipe Vanilla Installer
Building module [init-setup] of type [shell]
Module [init-setup] built successfully
Building module [firmware] of type [apt]
Loading new plugin
Using plugin: apt
Module [firmware] built successfully
Building module [input-and-locale] of type [apt]
Using plugin: apt
Module [input-and-locale] built successfully
Building module [] of type []
Loading new plugin
panic: /usr//share/vib/plugins/.so: cannot open shared object file: No such file or directory

goroutine 1 [running]:
github.com/vanilla-os/vib/core.LoadPlugin({0x0, 0x0}, 0x0, 0xc0000c02c0)
	/home/runner/work/Vib/Vib/core/plugins.go:33 +0x616
github.com/vanilla-os/vib/core.LoadBuildPlugin({0x0, 0x0}, {0xa3ca40, 0xc00013c660}, 0xc0000c02c0)
	/home/runner/work/Vib/Vib/core/plugins.go:81 +0xd7
github.com/vanilla-os/vib/core.BuildModule(0xc0000c02c0, {0xa3ca40, 0xc00013c660})
	/home/runner/work/Vib/Vib/core/build.go:420 +0x2e5
github.com/vanilla-os/vib/core.BuildModules(0xc0000c02c0, {0xc00025c000?, 0x242?, 0x1b6?})
	/home/runner/work/Vib/Vib/core/build.go:370 +0x115
github.com/vanilla-os/vib/core.BuildContainerfile(0xc0000c02c0)
	/home/runner/work/Vib/Vib/core/build.go:80 +0x173
github.com/vanilla-os/vib/core.BuildRecipe({0x7ffc91abcd3f?, 0xd549bd9300000000?})
	/home/runner/work/Vib/Vib/core/build.go:49 +0xdd
github.com/vanilla-os/vib/cmd.buildCommand(0xc0000f0400?, {0xc00003ae50?, 0x4?, 0xae766e?})
	/home/runner/work/Vib/Vib/cmd/build.go:80 +0x17e
github.com/spf13/cobra.(*Command).execute(0xc0001aa608, {0xc00003ae10, 0x1, 0x1})
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:985 +0xaaa
github.com/spf13/cobra.(*Command).ExecuteC(0xfca800)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:1117 +0x3ff
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:1041
github.com/vanilla-os/vib/cmd.Execute()
	/home/runner/work/Vib/Vib/cmd/root.go:56 +0x1a5
main.main()
	/home/runner/work/Vib/Vib/main.go:12 +0xf
Error: Process completed with exit code 2.
```

### Steps to Reproduce

try to build the image and watch it fail :)

### On what version of Vanilla OS this happens?

2.0

### Additional Information

https://github.com/jardon/installer-image/actions/runs/12038049630/job/33562833887

--- a/recipe.yml
+++ b/recipe.yml
@@ -32,9 +32,9 @@ stages:
       - name: packages-modules
         type: includes
         includes:
+          - gh:Vanilla-OS/core-image:main:modules/01-kernel.yml
           - gh:Vanilla-OS/core-image:main:modules/05-firmware.yml
           - gh:Vanilla-OS/core-image:main:modules/10-input-and-locale.yml
-          - gh:Vanilla-OS/core-image:main:modules/130-kernel.yml
           - gh:Vanilla-OS/core-image:main:modules/50-fs.yml
           - gh:Vanilla-OS/core-image:main:modules/70-compression.yml
           - gh:Vanilla-OS/core-image:main:modules/90-network.yml

